DorisProject/src/app/pages/stock/stock.component.ts

```diff
@@ -70,4 +74,22 @@ export class StockComponent implements OnInit {
   }
 
 
+  openModal(request: string) {
+    const dialogRef = this.dialog.open(ModalComponent, {
+      width: '445px', height: '235px',
```

Comment:
Codacy has a fix for the issue: [Insert `⏎·····`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
      width: '445px',
```